WasatchNET/Driver.cs

```diff
@@ -271,54 +271,58 @@ void logDevice(UsbRegistry usbRegistry)
             foreach (string line in deviceInfoSummaries)
                 logger.debug("Summary: {0}", line);
 
-            foreach (UsbConfigInfo cfgInfo in device.Configs)
+            // YOU ARE HERE -- does this die on Zynq? On ARM in general?
+            if (usbRegistry.Pid != 0x4000)
             {
-                logger.debug("Config {0}", cfgInfo.Descriptor.ConfigID);
-
-                // log UsbConfigInfo
-                logNameValuePairs(cfgInfo.ToString().Split(new char[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries));
-
-                // log UsbInterfaceInfo 
-                foreach (UsbInterfaceInfo interfaceInfo in cfgInfo.InterfaceInfoList)
+                foreach (UsbConfigInfo cfgInfo in device.Configs)

```

Comment:
I don't remember making this change.  This may be merging changes I did with changes Trent did.  I may need to just revert this whole file to what's on master.